equinox-shipping/Domain/Container.fs

```diff
@@ -0,0 +1,57 @@
+module Container
+
+// Container state needs to be serializable as it will be stored as part of the
```

Comment:
FYI some people take to doing a `module Types` for these sorts of shared things (without AutoOpen)
I personally don think that makes sense here

Reply:
Moved states to `Types.fs`. Thanks!